app/models/content_configuration.rb

```diff
@@ -31,6 +31,22 @@ class ContentConfiguration < Spree::Preferences::FileConfiguration
   preference :group_signup_case_studies_html, :text, default: I18n.t(:content_configuration_case_studies)
   preference :group_signup_detail_html, :text, default: I18n.t(:content_configuration_detail)
 
+  # Main URLs
+  preference :menu_1, :boolean, default: true
+  preference :menu_1_icon_name, :string, default: ""
```

Comment:
Do we need some default here? I guess in most instances, the fallback locale will be `en` which always contains values. Just if somebody configured a different fallback locale and hasn't completed the translation yet, this could be a problem. That sounds very unlikely. So I guess this can stay as is.